philistine/mne/io.py

```diff
@@ -43,42 +43,44 @@ def write_raw_brainvision(raw, vhdr_fname, events=True):
 
     Notes
     -----
-    The BrainVision format supports by-channel filter and measurement information and moreover distinguishes between hardware and software filters. MNE does neither.
-    Currently, filter information is not exported.
-
-    Moreover BrainVision also allows for more complex trigger codes than MNE's simple
-    integers, e.g. distinguishing on supported hardware between stimulus codes (prefixed by
-    an S) and responses codes (prefixed by an R). MNE's numeric events are all treated as
-    'stimulus markers' and prefixed by an S on output.
-
-    In other words, a round trip import-export is a lossy operation in terms of metadata.
-    The actual EEG recording should be losslessly preserved within the realm of floating
-    point precision.
+    The Brainvision format supports by-channel filter and measurement
+    information and moreover distinguishes between hardware and software
+    filters. MNE does neither. Currently, filter information is not exported.
+
+    Moreover Brainvision also allows for more complex trigger codes than MNE's
+    simple integers, e.g. distinguishing on supported hardware between stimulus
+    codes (prefixed by an S) and responses codes (prefixed by an R). MNE's
+    numeric events are all treated as 'stimulus markers' and prefixed by an S
+    on output.
+
+    In other words, a round trip import-export is a lossy operation in terms of
+    metadata. The actual EEG recording should be losslessly preserved within
+    the realm of floating point precision.
     """
 
     vmrk_fname = vhdr_fname[:-4] + 'vmrk'
     eeg_fname = vhdr_fname[:-4] + 'eeg'
 
     if isinstance(events, np.ndarray):
         pass
-    elif events == False:
-        events = np.ndarray([0,3])
-    elif events == True:
-        events = mne.find_events(raw,verbose=False)
+    elif not events:
```

Comment:
The explicit check against the boolean values was a stylistic decision because this parameter can have different types. Admittedly, the correct check would be with `is` (i.e. for identity and not equivalence) because things like empty lists and strings evaluate to `False`. In the other future, we might want to expand this argument to take string or list types and the empty ones might have a special meaning.